releases/v21.1.13.md

```diff
@@ -0,0 +1,135 @@
+- Manually enqueueing ranges via the DB Console will no longer crash nodes that contain an uninitialized replica for the enqueued range. [#73038][#73038]
+- Fixed a crash with message "attempting to propose command writing below closed timestamp" that could occur, typically on overloaded systems experiencing non-cooperative lease transfers. [#73166][#73166]
+- The `txnwaitqueue.pusher.waiting` metric no longer over-reports the number of pushing transactions in some cases. [#71743][#71743]
+- Fixed a rare condition that could cause a range merge to get stuck waiting on itself. The symptom of this deadlock was a goroutine stuck in `handleMergeInProgressError` for tens of minutes. [#72049][#72049]
+- Fixed bugs causing [`CREATE TABLE AS`](../v21.1/create-table-as.html) and [`CREATE MATERIALIZED VIEW`](../v21.1/create-view.html) to panic if the `SELECT` query was an internal table requiring internal database state. [#73620][#73620]
+- Fixed a rare internal error, "estimated row count must be non-zero", which could occur when planning queries using an inverted index. This error could occur if the histogram on the inverted index showed that there were no rows. [#73353][#73353]
+- Fixed an internal error, "empty Datums being compared to other", that could occur during planning for some `SELECT` queries over tables that included a `DEFAULT` partition value in a `PARTITION BY LIST` clause. This bug had been present since 21.1.0. The bug does not exist on versions 20.2.x and earlier. [#73663][#73663]
+- Fixed a bug in database and schema restore cleanup that resulted in a dangling descriptor entry on job failure. [#73412][#73412]
+- Fixed a bug with the ungraceful shutdown of distributed queries in some rare cases. [#73959][#73959]
+- Fixed a bug causing CockroachDB to return a spurious "context canceled" error for a query that actually succeeded in extremely rare cases. [#73959][#73959]
+- Fixed a bug which caused corruption of [partial indexes](../v21.2/partial-indexes.html), which could cause incorrect query results. The bug was only present when two or more partial indexes in the same table had identical `WHERE` clauses. This bug had been present since version 21.1.0. [#74475][#74475]
+
+### Performance improvements
+
+- Improved [`IMPORT INTO`](../v21.1/import.html) performance in cases where it encounters large numbers of unresolved write intents. [#72272][#72272]
+- The performance of transaction deadlock detection is now more stable even with significant transaction contention. [#71743][#71743]
+- Bulk ingestion of small write batches (e.g., index backfill into a large number of ranges) is now throttled, to avoid buildup of read amplification and associated performance degradation. Concurrency is controlled by the new cluster setting `kv.bulk_io_write.concurrent_addsstable_as_writes_requests`. [#74072][#74072]
+
+### Doc updates
```

Comment:
@ericharmeling If there are no doc updates, for example, should we not remove the section altogether?

Reply:
Thank you. Yes.

Done.